tests/integration_tests.py

```diff
@@ -377,14 +390,14 @@ def build_test_list():
             [
                 [
                     "--checkpoint.enable_checkpoint",
-                    "--experimental.pipeline_parallel_degree 2",
-                    "--training.enable_cpu_offload True",
-                    "--optimizer.early_step_in_backward",
+                ],
+                [
+                    # placeholder for the generation script's generate step
```

Comment:
is this part WIP?

Reply:
No, I wrote it this way because for generation, the first step is to call `run_llama_train.sh` and the second step is to call the generating script
https://github.com/pytorch/torchtitan/pull/741/files#diff-3b751e36d12b5fa68ae66727b4d8c6ef2cce12d4f2444b46fd882942c9f4a87fR441-R447

To make it less hacky I think we need some surgery to the file & classes.